src/event_button.rs

```diff
@@ -33,4 +33,14 @@ impl EventButton {
     pub fn get_device(&self) -> Option<::Device> {
         unsafe { from_glib_none(self.as_ref().device) }
     }
+
+    pub fn get_axes(&self) -> Option<(f64, f64)> {
+        let axes = self.as_ref().axes;
+
+        if axes.is_null() {
+            None
+        } else {
+            unsafe { Some((*axes, *axes.offset(1))) }
+        }
+    }
```

Comment:
Please check the docs and add *all* the missing getters while we're at it :)

There's at least `x_root` and `y_root` still missing here and maybe other things for the other events

Reply:
Damn, I thought it was implemented in `Event`... I'll need to make a bigger check.